app/models/colyseus-models/pokemon.ts

```diff
@@ -6565,7 +6565,7 @@ export class Ninjask extends Pokemon {
 export class Shedinja extends Pokemon {
   types = new SetSchema<Synergy>([Synergy.BUG, Synergy.GHOST, Synergy.FLYING])
   rarity = Rarity.EPIC
-  stars = 2
+  stars = 0
```

Comment:
this is not the correct way to solve the issue, Shedinja is a tier 2

if you want to change sell price of Shedinja, do it in getSellPrice() function

We could also fix the issue by considering Shedinja/Munja a Pokemon duo. We already have special rules for sell prices of unique duos